packages/leap/lib/src/leap_game.dart

```diff
@@ -1,58 +1,62 @@
 import 'dart:ui';
 
+import 'package:flame/components.dart';
 import 'package:flame/game.dart';
-import 'package:leap/leap.dart';
-import 'package:leap/src/utils/has_tracked_components.dart';
-import 'package:leap/src/utils/lifecycle_state_aware.dart';
+import 'package:leap/src/entities/entities.dart';
+import 'package:leap/src/leap_map.dart';
+import 'package:leap/src/leap_world.dart';
+import 'package:leap/src/mixins/mixins.dart';
 
-/// A Flame game with all the Leap built-ins
+/// A [FlameGame] with all the Leap built-ins.
 class LeapGame extends FlameGame with HasTrackedComponents {
-  late LeapMap map;
-  late LeapWorld world;
+  LeapGame() : appState = AppLifecycleState.resumed;
 
-  AppLifecycleState appState = AppLifecycleState.resumed;
+  late final LeapMap map;
+  late final LeapWorld leapWorld;
```

Comment:
We shouldn't need a `leapWorld` field, we can set `world = LeapWorld()` in the `super(world: LeapWorld)` constructor, it just means we have to refactor the leap world constructor a bit so that the gravity re-calculated when `tileSize` is set instead of `onLoad`